### PR TITLE
v1.1.3: display memory usage and runtime when AC

### PR DESCRIPTION
**Pull request by dlwocks31**

### CODE REVIEW

**Comment by sohnryang** [comment]
오 멋지네요

**Comment by sohnryang** [comment]
잠만 테스트해볼게요

**Comment by sohnryang** [comment]
Would be nice to add spaces.
Current version shows something like `AC(1984KB, 0ms)`, but `AC (1984KB, 0ms)` would be better.

File: boj/__init__.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def convert_msg(msg):
         msg = Fore.YELLOW + 'Partial ({0})'.format(re.match(r'\d+', msg)[0])
         return msg + Style.RESET_ALL
     conversion_table = {
-        '맞았습니다!!': Fore.GREEN + 'AC',
+        '맞았습니다!!': Fore.GREEN + 'AC(%sKB, %sms)' % (mem, rtime),
```
Comment (sohnryang) [inline boj/__init__.py]: `'AC(%sKB, %sms)'` should be `'AC (%sKB, %sms)'` for readability.

**Review by sohnryang — CHANGES_REQUESTED** [review]

File: boj/__init__.py (hunk 1 above)
Comment (dlwocks31) [reply boj/__init__.py]: 고쳤어요~

**Pull request merged**